dnf-behave-tests/features/remove-protected.feature

```diff
@@ -0,0 +1,12 @@
+Feature: Do not remove protected RPMs
+
+# should fail until dnf-4.2.1 is on RHEL8
+@xfail 
```

Comment:
The scenario is already passing on RHEL, please remove @xfail and let's merge it.